webapp/webapp/demo/urls.py

```diff
@@ -5,5 +5,5 @@
 urlpatterns = [
     path('', train_annotations, name='train_annotations'),
     path('auth_callback', validate_umls_user, name='validate_umls_user'),
-    path('download', download, name="download")
+    path('download_model', download_model, name="download_model")
```

Comment:
not critical but generally should use dashes for urls
 https://sitechecker.pro/site-audit-issues/pages-underscores-dashes-url/

Reply:
Changed to hyphenated urls. Not sure SEO applies to this use case though.

Reply:
thanks - not just for SEO - its generally best practise: https://www.thecodebuzz.com/restful-api-url-naming-conventions-best-practices/